### DISCUSSION TITLE
[@types/ssh2]: missing utils

### DISCUSSION THREAD

**Discussion by idan-at** [DefinitelyTyped/DefinitelyTyped#59320] [Issues with a @types package]
Hey,

As you can see [here](https://github.com/mscdex/ssh2/blob/master/lib/index.js#L34), `ssh2` exports several properties under the `utils` key.
However, in the types, [utils is exported](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ssh2/index.d.ts#L39), which only contains the `parseKey` helper ([here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ssh2-streams/index.d.ts#L1707))

[a comment by an automated account is not shown]

**Comment by bragle** [comment]
PR welcome 😸

**Comment by MysteryBlokHed** [comment]
I just opened #59326 to fix this

**Reply by MysteryBlokHed** [reply]
PR was just merged
